src/com/sap/piper/DownloadCacheUtils.groovy

```diff
@@ -3,7 +3,35 @@ package com.sap.piper
 
 class DownloadCacheUtils {
 
+    static Map injectDownloadCacheInMavenParameters(Script script, Map parameters) {
+        if (DownloadCacheUtils.isEnabled(script)) {
+
+            if (!parameters.dockerOptions) {
+                parameters.dockerOptions = []
+            }
+            if (parameters.dockerOptions instanceof CharSequence) {
```

Comment:
why CharSequence and not String?

Reply:
We had a Jenkins Runtime Exception earlier, because something which should have been a String was in fact something similar. This is more generic. I can't remember where we encountered this problem...

Reply:
But one example where the code would behave more correct would be GString. You still want a list in the end.